Source/BugsnagConfiguration.m

```diff
@@ -218,9 +216,9 @@ - (instancetype _Nonnull)initWithApiKey:(NSString *_Nonnull)apiKey
     // persistUser isn't settable until post-init.
     _user = [self getPersistedUserData];
     [self setUserMetadataFromUser:_user];
-    
+
     #if !DEBUG
-        _enabledErrorTypes |= BSGErrorTypesOOMs;
+        _enabledErrorTypes.OOMs = true;
```

Comment:
Do we need this if you're setting it in the `BugsnagErrorTypes`'s `init` function too?

Reply:
nope, this can go.